### PR TITLE
modify kusion_deps in kusion/docs/cmd

### PR DESCRIPTION
**Pull request by Vikky-00**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., design docs, usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by xi-wen-kk** [comment]
recheck

**Comment by xi-wen-kk** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by SparkYuan** [comment]
@xi-wen-kk Welcome new contributors to KusionStack! 👏🏻👏🏻👏🏻

**Comment by healthjyk** [comment]
@xi-wen-kk Hi, Welcome to Kusion! You've found a very meaningful issue, the internationalization(i18n) of Kusion.

**Comment by healthjyk** [comment]
@xi-wen-kk Hi, Welcome to Kusion! You've found a VERY meaningful issue, the internationalization(i18n) of Kusion.
I have added some comments of your commits, plz take a view of them :)
Besides, maybe you should add more description of your PR, and add an issue of it. Kusion does not have enough Chinese translations, completing them is an important thing. And, how to do it automatically is also important, Kusion offers /scripts/gen-docs to generate doc from code, and i18n support in pkg/util/i18n/translations. You can use the script, edit file .pot, and then generate the doc :)

File: .github/workflows/test.yaml
[the remark this answers is not in the document]
Comment (xi-wen-kk) [reply .github/workflows/test.yaml:1]: sorry, I want to cancel my commit to this file, but deleted the file by mistake

[the remark this answers is not in the document]
Comment (healthjyk) [reply .github/workflows/test.yaml:1]: ok, just recover it :)

File: docs/cmd/zh/kusion_deps.md [hunk 1]
```diff
@@ -1,39 +1,38 @@
...
-  # List all the KCL files that are dependent by the given focus paths
+  # 列出所有依赖给定路径下的KCL文件
   kusion deps --focus path/to/focus1 --focus path/to/focus2
   
-  # List all the projects that depend on the given focus paths
+  # 列出所有依赖给定路径的项目
   kusion deps --direct down --focus path/to/focus1 --focus path/to/focus2
   
-  # List all the stacks that depend on the given focus paths
+  # 列出所有依赖给定路径的堆栈
```
[the remark this answers is not in the document]
Comment (xi-wen-kk) [reply docs/cmd/zh/kusion_deps.md:21]: ok, got it.

**Pull request closed without merging**